### DISCUSSION TITLE
Wofür ist ARC4 und wo finde ich das?

### DISCUSSION THREAD

**Discussion by Conan179** [Freetz-NG/freetz-ng#1071] [Allgemein] [answered]
Hallo erst mal.
Ich wollte für meine 6690, mit dem Fritzos 8.00 ein neues Image bauen.
Es bricht beim Step 2 mit dieser meldung ab:
installing and stripping modules
  cifs.ko ...........................  191.24 Kb
WARNING: File of kernel module not found: arc4.ko
ERROR: Enable missing with 'make kernel-menuconfig' as (M)odule (and push a pull request). Use '/' to search in menuconfig.
FINISHED
Meine Frage ist, was ist arc4 und kannst du dir evtl. einen reim draus machen, warum es jetzt erst meckert? Ich hab die selbe .config wie beim Fritzos 7.5x, natürlich nach "git pull", "make menuconfig" gespeichert und dann make.

**Comment by aus-sua** [comment]
Wie in der Error-Erläuterung "angesagt", mit `make kernel-menuconfig` als M markieren...
Die Linux Kernel Configuration ist gewöhnungsbedürftig zu bedienen und zu durchsuchen, 
die arc4.ko sollte darin hier zu "(M)odul-ieren" sein:
![arc4 ko_](https://github.com/user-attachments/assets/d172f058-be76-4a5d-9b43-686608268105)

Unabhängig davon (die Modul-Auswahl wirst Du wohl auch bei einer 
neuen, cleanen .config erstellen müssen):  
Es ist wirklich keine gute Idee, die .config von der alten zur neuen Firmware-Version 
"nur faul rüber zu kopieren" (bitte unbedingt die freetz-Hilfe/-Doku dazu lesen!)...
Erstelle sie lieber neu! :-)

Wenn es läuft, dann erstelle bitte daraus hier ein github-"Pull request", das kommt dann auch
den anderen freetz-NG-Nutzern zugute!
:
Oder eben kurz gesagt, 
s.o.:   "Enable missing with 'make kernel-menuconfig' as (M)odule (and push a pull request)" 
;-)

**Comment by Conan179** [comment]
In dem menü gibt es keine option zum aktivieren.

**Comment by fda77** [comment, marked as answer]
sollte gehen
